JuiceMaker/JuiceMaker/Model/JuiceMaker.swift

```diff
@@ -7,13 +7,11 @@ struct JuiceMaker {
         let juiceRecipe = order.recipe
         
         do {
-            try fruitStore.isEnoughStock(for: juiceRecipe)
-            fruitStore.decreaseFruitStock(by: juiceRecipe)
+            try fruitStore.decreaseFruitStock(by: juiceRecipe)
             print("\(order)가 완성되었습니다")
-        } catch JuiceMakeError.outOfStock {
-            print("과일 재고가 부족해서 쥬스 제작 불가")
-        } catch {
-            print("알 수 없는 문제 : \(error)")
+        } catch let error {
+            print("\(error.localizedDescription)")
         }
```

Comment:
👍 
error는 기본으로 제공해주므로 따로 캐스팅이 필요없다면,
```suggestion
        } catch {
            print("\(error.localizedDescription)")
        }
```
으로도 표현 가능합니다 🙂